sui/gateway.js

```diff
@@ -106,11 +107,11 @@ function getProof(keypair, commandType, data, contractConfig, options) {
 }
 
 async function callContract(keypair, client, config, chain, args, options) {
-    if (!chain.contracts.axelar_gateway) {
+    if (!chain.contracts.AxelarGateway) {
         throw new Error('Axelar Gateway package not found.');
     }
```

Comment:
Move this check to the `mainProcessor` instead and delete the duplicated ones in `rotateSigners`, `callContract`, and `approveMessages` functions.